PENDING_RELEASE_NOTES.md

```diff
@@ -7,5 +7,6 @@ _Date TBD_
 * Updated display of error messages when merging a developer to support multiple error messages
 * Show QMS standards under (g)(4) criteria details
 * Add "Scheduled Jobs" section for Quartz jobs
+* Add http:// to external urls so that they can be followed
```

Comment:
STG is frozen right now, and these RELEASE_NOTES sections were already deployed to PROD. This note should be in PENDING_PENDING_RELEASE_NOTES.